src/pages/confirm.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Layout from '../components/Layout'
+import get from 'lodash/get'
+
+class Confirm extends React.Component {
```

Comment:
this page thanks the new subscriber and asks them to confirm their email address for double opt-in.